styles.css

```diff
@@ -60,6 +60,38 @@ body {
   #directorTextDiv {
     padding: 0rem 2rem 0rem 2rem;
   }
+
+  #servicesHeading {
+    font-size: 3.5rem;
+  }
+
+  #servicesTextRow {
+    margin-top: -2.5rem;
+  }
+
+  #historyHeading {
+    text-align: left;
```

Comment:
we can just apply the bootstrap css class `text-left` to the element itself in the html file instead of defining the style here in this css file.

i noticed you have two different text align style for this `historyHeading` based on screen size.

bootstrap also provided these classes for media query. so you can also apply these two `text-sm-left text-md-right`. 
`text-sm-left` will left aligned text on viewports sized SM (small) or wider. 
`text-md-right` will right aligned text on viewports sized MD (medium) or wider